requirements/common.in

```diff
@@ -41,6 +41,6 @@ importlib-metadata
 dockerflow
 
 # Measuring noise of perf data
-measure-noise
+moz-measure-noise
```

Comment:
@jmaher do you know if we can pull packages from our internal pypi mirror here?

Reply:
I don't think it is possible.  Treeherder is an external app, although it might be on the same network to access internal pypi.